LICENSE

```diff
@@ -1,8 +1,6 @@
-nbgrader is licensed under the terms of the Modified BSD License
-(also known as New or Revised or 3-Clause BSD), as follows:
-
-Copyright (c) 2014, Jupyter Development Team
+BSD 3-Clause License
 
+Copyright (c) 2017, Project Jupyter Contributors
```

Comment:
Before merging, we also need to change date back to 2014. The date should be the year of first copyright, not last. 2014  is implicitly 2014-present.